### PR TITLE
[DO NOT MERGE] demonstrate commands in a bash script to help

### PR DESCRIPTION
**Pull request by deads2k**
Something gets lost in translation from instructions here https://github.com/openshift/origin/pull/14285.  To help, I've created a bash script which works against a built master branch.  I still think you'll need to learn about the certificates already in play in the master-config.yaml, but maybe this will help.

Just tried it on my machine, works fine.

@jpeeler @pmorie @derekwaynecarr

### CODE REVIEW

**Comment by jpeeler** [comment]
The script works for me too.

**Comment by deads2k** [comment]
> The script works for me too.

great

File: hack/enable-aggregator.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+rm -rf openshift.local.config/master
+
+openshift start --write-config=openshift.local.config
+
+mv front-proxy-ca.crt openshift.local.config/master/
+mv aggregator-front-proxy.crt openshift.local.config/master/
+mv aggregator-front-proxy.key openshift.local.config/master/
+
+sed -i 's|certFile: ""|certFile: aggregator-front-proxy.crt|g' openshift.local.config/master/master-config.yaml
```
Comment (stevekuznetsov) [inline hack/enable-aggregator.sh]: didn't we add `config patch` for this?

Comment (stevekuznetsov) [inline hack/enable-aggregator.sh]: Oh wait, not for merging.

**Pull request closed without merging**